parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -211,9 +211,12 @@ pub mod pallet {
 				}
 			})?;
 
+			// Convert to a sibling parachain ID (uses bytes "sibl" instead of "para" to derive
+			// the sovereign account ID, to match how Assethub derives the account ID).
+			let sibling_para_id: SiblingParaId = envelope.dest.into();
 			// Reward relayer from the sovereign account of the destination parachain
 			// Expected to fail if sovereign account has no funds
-			let sovereign_account = envelope.dest.into_account_truncating();
+			let sovereign_account = sibling_para_id.into_account_truncating();
```

Comment:
We need to do the same conversion in this pallet as well as the control pallet, so lets make the converter a pluggable type.

```suggestion
			let sovereign_account = T::SiblingSovereignAccountOf::convert(para_id)
```

where `T::SiblingSovereignAccountOf` is a type that implements [Convert<ParaId, T::AccountId>](https://paritytech.github.io/polkadot-sdk/master/sp_runtime/traits/trait.Convert.html#tymethod.convert)

Reply:
Actually, wait the above is totally overkill. Instead we can simplify by adding a free function in snowbridge core:

```rust
fn sibling_sovereign_account<T>(para_id: ParaId) -> T::AccountId
where
   T: frame_system::Config
{
    SiblingParaId::from(para_id).into_account_truncating()
}
```